internal/lpastore/lpa.go

```diff
@@ -484,6 +486,26 @@ func FromDonorProvidedDetails(l *donordata.Provided) *lpadata.Lpa {
 		}
 	}
 
+	var authorisedSignatory lpadata.Actor
+	if v := l.AuthorisedSignatory; v.FirstNames != "" {
+		authorisedSignatory = lpadata.Actor{
+			// TODO: add UID for this actor
```

Comment:
I'm leaving these TODOs so we remember (hopefully) when we get to sending this data to the lpa-store